### PR TITLE
Luk10114/1145 view point cloud data offline

### PR DESCRIPTION
**Pull request by LDuncAndroid**

### CODE REVIEW

File: java/view-point-cloud-data-offline/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# View point cloud data offline
+
+Display local 3D point cloud data.
+
+![View point cloud data offline](view-point-cloud-data-offline.png)
+
+## Use Case
```
Comment (TADraeseke) [inline java/view-point-cloud-data-offline/README.md]: sentence case section titles

File: kotlin/view-point-cloud-data-offline/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# View point cloud data offline
+
+Display local 3D point cloud data.
+
+![View point cloud data offline](view-point-cloud-data-offline.png)
+
+## Use Case
```
Comment (TADraeseke) [inline kotlin/view-point-cloud-data-offline/README.md]: as above

File: kotlin/view-point-cloud-data-offline/src/main/java/com/esri/arcgisruntime/samples/viewpointclouddataoffline/MainActivity.kt [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                Surface().apply {
+                    elevationSources.add(ArcGISTiledElevationSource(getString(R.string.elevation_source_url)))
+                }.let { surface ->
+                    // set the base surface of the scene
+                    this.baseSurface = surface
+                }
+            }
+        }
+
+        // For API level 23+ request permission at runtime
```
Comment (TADraeseke) [inline kotlin/view-point-cloud-data-offline/src/main/java/com/esri/arcgisruntime/samples/viewpointclouddataoffline/MainActivity.kt]: as above

File: kotlin/view-point-cloud-data-offline/src/main/java/com/esri/arcgisruntime/samples/viewpointclouddataoffline/MainActivity.kt [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            createPointCloudLayer()
+        } else {
+            // report to user that permission was denied
+            Toast.makeText(this, resources.getString(R.string.read_permission_denied_message),
+                    Toast.LENGTH_SHORT).show()
+        }
+    }
+
+    private fun createPointCloudLayer() {
+        // Add a point cloud layer to the scene by passing the URI of the Scene Layer Package to the constructor
```
Comment (TADraeseke) [inline kotlin/view-point-cloud-data-offline/src/main/java/com/esri/arcgisruntime/samples/viewpointclouddataoffline/MainActivity.kt]: as above, stick to all lower case comments

File: java/view-point-cloud-data-offline/src/main/java/com/esri/arcgisruntime/samples/viewpointclouddataoffline/MainActivity.java [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+      } else if (loadStatusChangedEvent.getNewLoadStatus() == LoadStatus.FAILED_TO_LOAD) {
+        // Notify user that the PointCloudLayer has failed to load
+        Toast.makeText(this, R.string.point_cloud_layer_load_failure_message, Toast.LENGTH_LONG).show();
+      }
+    });
+
+    // Load the PointCloudLayer asynchronously
+    pointCloudLayer.loadAsync();
+  }
+}
```
[the remark this answers is not in the document]
Comment (LDuncAndroid) [reply java/view-point-cloud-data-offline/src/main/java/com/esri/arcgisruntime/samples/viewpointclouddataoffline/MainActivity.java]: Had been added in https://github.com/Esri/arcgis-runtime-samples-android/pull/584/commits/6679a27eb54f1575cf6bf42b2dba842805fdd83f

**Comment by LDuncAndroid** [comment]
Thanks for reviewing @TADraeseke . Everything should be resolved.

**Pull request merged**